### PR TITLE
docs: initial draft with many TODOS

### PR DESCRIPTION
**Pull request by dirien**
initial draft with many TODOS.

Looking forward to fill as many points as possible. Maybe we have the informations already, like project governance etc. And we need just to bring the pieces together.

Looking forward.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,36 @@
...
+## Participate and contribute
+
+### Monthly Community meeting and release planning
+
+We maintain a XXX with both our community calls and release planning meeting series.
+
+Join us in our monthly community meetings to engage with the GoReleaser community interactively, to learn about the
+project's directions, and to discuss contributions. Meeting agendas are published in advance - watch out for open PRs on
+the community-meetings/ folder. If you have a pressing issue of common interest you'd like discussed, please let us know
+on Matrix, or simply bring it to the next meeting's Q&A.
```
Comment (caarlos0) [inline README.md]: maybe change matrix to discord (and link an invite here?)

**Review by caarlos0 — COMMENTED** [review]
lgtm overal

**Review by radeksimko — COMMENTED** [review]
Regarding the CoC - I believe that on GitHub one can upload the CoC once under `org-name/.github` and it should then be automatically assigned as a global CoC for all repositories under that org. Assuming we want to apply to same CoC - is it worth taking that approach instead of duplicating it across repos?

https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file

**Comment by caarlos0** [comment]
Yeah, that makes sense @radeksimko 

We even have the repo already: https://github.com/goreleaser/.github

Will create it there!

EDIT: done https://github.com/goreleaser/.github/blob/main/CODE_OF_CONDUCT.md

**Comment by crazy-max** [comment]
Perhaps we could also refer to or be inspired by the [GitHub Etiquette](https://github.com/kossnocorp/etiquette/blob/master/README.md)?

**Comment by caarlos0** [comment]
> Perhaps we could also refer to or be inspired by the [GitHub Etiquette](https://github.com/kossnocorp/etiquette/blob/master/README.md)?

currently we are using the contributor covenant, which is kind of standard it seems, not sure if its worth changing that as it covers pretty much everything already, and seems widely used https://www.contributor-covenant.org 

that said, I think its worth adding to issue templates something like "please be polite" et al, and link to it... does that makes sense? 🤔

File: README.md (hunk 1 above)
Comment (dirien) [reply README.md]: done

**Comment by dirien** [comment]
@caarlos0 and @radeksimko done!, thanks for hint.

**Review by caarlos0 — APPROVED** [review]

**Comment by dirien** [comment]
@caarlos0, any date idea for the first call?

**Comment by caarlos0** [comment]
I would say >mid-january... lets say... the January 21th?

**Pull request merged**